src/components/Dialog/ActionDialog/ActionDialogContentInner.tsx

```diff
@@ -26,7 +27,7 @@ export type BaseProps = {
    */
   subtitle?: ReactNode
   /**
-   * ダイアログタイトルの HTML タグ
+   * @deprecated titleTag属性は非推奨です。SectioningContent(Article, Aside, Nav, Section, SectioningFragment)でDialog全体をラップして、ダイアログタイトルのHeadingレベルを設定してください
```

Comment:
nits: `@deprecated` 付けてるので、対処方法だけ案内でよさそう

```suggestion
   * @deprecated SectioningContent(Article, Aside, Nav, Section, SectioningFragment)でDialog全体をラップして、ダイアログタイトルのHeadingレベルを設定してください
```